homeassistant/components/risco/config_flow.py

```diff
@@ -158,6 +158,7 @@ async def async_step_local(self, user_input=None):
             try:
                 info = await validate_local_input(self.hass, user_input)
             except CannotConnectError:
+                _LOGGER.exception("Cannot connect")
```

Comment:
Normally we don't log the stack trace at error level if it's a known exception. I'd recommend logging the stack trace at debug level instead.

Reply:
Ok. I changed it here: https://github.com/home-assistant/core/pull/82730